brigade.js

```diff
@@ -92,7 +176,7 @@ class Notification {
   // Send a new notification, and return a Promise<result>.
   run() {
     this.count++;
-    var job = new Job(`${ this.name }-notification-${ this.count }`, "brigadecore/brigade-github-check-run:latest");
+    var job = new Job(`${ this.name }-notification-${ this.count }`, "brigadecore/brigade-github-check-run:edge");
```

Comment:
I know this is a conscious decision for this project to dogfood itself. You might want to add a comment here to that effect so that others who might emulate this in their own `brigade.js` are forewarned.

Reply:
Good call; comment added.